gpytorch/kernels/kernel.py

```diff
@@ -187,42 +187,28 @@ def forward(self, x1, x2, diag=False, batch_dims=None, **params):
         """
         raise NotImplementedError()
 
-    def _create_input_grid(self, x1, x2, diag=False, batch_dims=None, **params):
```

Comment:
If you're removing this function from Kernels, can you update the docstrings? (This function is reference to a lot)